pkgs/applications/blockchains/liana/default.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, rustPlatform
+, fetchFromGitHub
+, llvmPackages
+, fetchurl
+, pkg-config
+, copyDesktopItems
+, makeDesktopItem
+, systemd
+, cmake
+, pkgs
+, makeWrapper
+, xorg
+, expat
+, fontconfig
+, freetype
+, libGL
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "liana";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "wizardsardine";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-cqvjPa/GXDdFv8XEPJgHfKP/Duzv+jmp1yZ1rlKPfE8=";
+  };
+
+  cargoLock = {
+    lockFile = "${src}/gui/Cargo.lock";
+    outputHashes = {
+      "liana-1.1.0" = "sha256-ncy7/w9kqD3w+/v0OVKgtgCHCF7Ram327I8R6IJe/s8=";
+      "miniscript-9.0.0" = "sha256-5ITjVqh5frqjTTs993xqo5WwDZ5dDAfFn4DGqt2QbiY=";
+    };
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    cmake
+    copyDesktopItems
+    makeWrapper
+  ];
+
+  buildInputs = [
+    expat
+    fontconfig
+    freetype
+    freetype.dev
+    libGL
+    pkg-config
```

Comment:
Is this really needed in `buildInputs` as well? Can you try it without this?

Reply:
Oh yeah, not needed. And I moved the runtime libs into their own variable and made use of `lib.makeLibraryPath`.

Builds and runs fine still :)